.github/workflows/workspace-setup.yml

```diff
@@ -0,0 +1,86 @@
+name: Workspace Setup
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 10 * * SUN"
+
+concurrency:
+  group: ${{ github.ref_name }}-test-ws-setup
+
+jobs:
+  test:
+    runs-on: macos-12
+    timeout-minutes: 30 # MacOS runners are more expensive than linux, so we want to be sure to stop any stuck builds.
+    environment:
+      name: test-ws-setup
+    env:
+      STAGE_NAME: test-ws-setup
+    permissions:
+      id-token: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: ${{ secrets.AWS_OIDC_ROLE_TO_ASSUME }}
+          aws-region: us-east-1
+          role-duration-seconds: 10800
+
+      - name: Run workspace setup
+        run: |
+          sh docs/assets/setup.sh
+
+      - name: Run nvm install
+        run: |
+          cat /tmp/rcfile
+          source /tmp/rcfile
+          nvm install
```

Comment:
Since nvm install is part of workspace setup, shouldn't we make it do it as part of that test?